pushHandler.go

```diff
@@ -59,54 +63,69 @@ func (h *pushHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		user = parsedPayload.Repository.Owner.Login
 		repo = parsedPayload.Repository.Name
 	}
-	compID := componentID{user, repo, "test_hash"}
+
+	compID := worker.ComponentID{User: user, Repo: repo, Hash: "test_hash"}
```

Comment:
Can we get a comment that points to the issue? I didn't realize we had one

Reply:
issue #7